internal/slicer/slicer_test.go

```diff
@@ -893,11 +893,31 @@ func runSlicerTests(c *C, tests []slicerTest) {
 			archives[name] = archive
 		}
 
+		pkgArchives := map[string]archive.Archive{}
+		for _, pkg := range release.Packages {
```

Comment:
We are no longer testing the package sorting if we are adding it manually in the tests. Once we move the logic to slicer we can remove this block and test it directly.

Reply:
N/A, updated the tests.